src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -756,5 +816,7 @@ class SnykSettingsDialog(
 
     fun getCliReleaseChannel(): String = cliReleaseChannelDropDown.selectedItem as String
 
+    fun isNetNewIssuesSelected(): String = netNewIssuesDropDown.selectedItem as String
```

Comment:
shouldn't that be `getNetNewIssuesSelection()`? `Is..` are usually returnin boolean values.

Reply:
Fixed.

Reply:
Fixed!